src/platform/plugins/shared/dashboard/server/plugin.ts

```diff
@@ -133,6 +136,17 @@ export class DashboardPlugin
   public start(core: CoreStart, plugins: StartDeps) {
     this.logger.debug('dashboard: Started');
 
+    if (plugins.share) {
+      plugins.share.url.locators.create(
+        new DashboardAppLocatorDefinition({
+          useHashedUrl: false,
```

Comment:
If we want to support hashed urls on the server, we'll need the request context so we can scope the server-side `UISettingsServiceStart.asScopedToClient` to retrieve the `'state:storeInSessionStorage'` setting as we do in the client.